src/main/java/core/basesyntax/Zoo.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Zoo {
+
+    private final List<LandAnimals> cellForLandAnimals = new ArrayList<>();
+    private final List<Bird> cellForBirds = new ArrayList<>();
```

Comment:
```suggestion
    private final List<Flyable> cellForBirds = new ArrayList<>();
```
we are interested not in birds but in animals, who can fly

Reply:
changed